homeassistant/components/lupusec/__init__.py

```diff
@@ -73,10 +75,50 @@ def setup(hass: HomeAssistant, config: ConfigType) -> bool:
     return True
 
 
+async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
+    """Set up this integration using YAML is not supported."""
+    return True
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up this integration using UI."""
+
+    if hass.data.get(DOMAIN) is None:
+        hass.data.setdefault(DOMAIN, {})
+
+    host = entry.data.get(CONF_HOST)
+    username = entry.data.get(CONF_USERNAME)
+    password = entry.data.get(CONF_PASSWORD)
+    name = entry.data.get(CONF_NAME)
+
+    try:
+        lupusec_system = await hass.async_add_executor_job(
+            LupusecSystem, username, password, host, name
+        )
+        hass.data[DOMAIN] = lupusec_system
+    except LupusecException as ex:
+        _LOGGER.error(ex)
+
+        persistent_notification.create(
+            hass,
+            f"Error: {ex}<br />You will need to restart hass after fixing.",
+            title=NOTIFICATION_TITLE,
+            notification_id=NOTIFICATION_ID,
+        )
+        return False
+
+    for platform in LUPUSEC_PLATFORMS:
+        hass.async_add_job(
+            hass.config_entries.async_forward_entry_setup(entry, platform)
+        )
```

Comment:
You can just pass a list to the setup.


```suggestion
    await hass.config_entries.async_forward_entry_setup(entry, PLATFORMS)
```

Reply:
Changed to `async_forward_entry_setup**s**`